frontend/src/app/ReportingApp.tsx

```diff
@@ -83,6 +83,18 @@ const App = () => {
     if (!data) return;
 
     appInsights?.setAuthenticatedUserContext(data.whoami.id, undefined, false);
+    if (window?.visualViewport?.width) {
+      appInsights?.trackMetric(
+        {
+          name: "userViewport_reporting",
+          average: window?.visualViewport?.width,
```

Comment:
Do we need the `?`s here since we are checking it above?

Reply:
probably not... I will remove it!

Reply:
ok thank you!!! :D